core/ir/aarch64/instr_create.h

```diff
@@ -2514,5 +2520,28 @@
 #define INSTR_CREATE_umull2_vector(dc, Rd, Rm, Rn, width) \
     instr_create_1dst_3src(dc, OP_umull2, Rd, Rm, Rn, width)
 
+/**
+ * Creates a DMB instruction.
+ * \param dc      The void * dcontext used to allocate memory for the instr_t.
+ * \param imm     The integer constant opnd_t operand.
```

Comment:
Point at one of the DR_DMB_OSHLD values, using `#` to make a doxygen link, so the user knows there are named constants.

Reply:
Done in the latest commit, but the Doxygen is not generated correctly.